packages/fulfillment/src/services/fulfillment-module-service.ts

```diff
@@ -42,6 +47,7 @@ const generateMethodForModels = [
   ShippingProfile,
   ShippingOptionRule,
   ShippingOptionType,
+  // Not adding Fulfillment to not auto generate the methods under the hood and only provide the methods we want to expose8
```

Comment:
question: I can't remember, what methods do we not want to expose?

nitpicking: Also, minor typo in the last word -> `expose8`

Reply:
😂 thanks, we don't want createFulfillments, deleteFulfillments, softDeleteFulfillments etc that are all bulk and we agreed on non supporting bulk operation when it involves external providers

Reply:
That's right. Thanks for reminding me. I'll have to update payment sessions to be in line with this.